blockchain/state_manager/src/lib.rs

```diff
@@ -117,6 +131,98 @@ where
         Ok((state_root, rect_root))
     }
 
+    pub fn tipset_state(&mut self, tipset: &Tipset) -> Result<(Cid, Cid), Box<dyn StdError>> {
+        trace!("tipSetState");
```

Comment:
maybe at least rename to our function casing? would be nice to trace the tipset Cid which state is being generated, this log is generally pretty useless